rust/parquet/src/util/test_common/page_util.rs

```diff
@@ -155,3 +157,21 @@ impl DataPageBuilder for DataPageBuilderImpl {
         }
     }
 }
+
+pub struct InMemoryPageReader {
```

Comment:
Do we need to document this test page reader?

Reply:
Fixed.